tox.ini

```diff
@@ -0,0 +1,29 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
```

Comment:
From your last comment it looks like Travis will be running `pytest` now @cfculhane? Do we still need `tox.ini`?

Reply:
Yeah I think tox is easier for end users, up to you though. I like to use tox to test fresh installs, and it's easy to test across multiple environments as it creates its own tox venv's for each python version.